backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -54,6 +75,58 @@ def insert_user_recommendation(args):
         modification_date__lt=(datetime.now() + timedelta(hours=-2))
     ).delete()
 
+def select_recommendation(args):
+    """Get Recommendations"""
+
+    request_user = args[constants.USER]
+    page_number = 1 if constants.PAGE_NUMBER not in args else int(args[constants.PAGE_NUMBER])
+
+    recommendation_queryset = UserRecommendation.objects.filter(user_id=request_user.id)
+
+    recommendations = get_results_from_queryset(recommendation_queryset, 5, page_number)
```

Comment:
Why did you think 5 is a proper number?

Reply:
It is just a temporary thing. But I think when the main feed is consisted of 20 cards, more than 5 is not so good.

Reply:
Do you consider the way that the Main Page calls two APIs for subscription and recommendation, and weave the results somehow for spreading cards, don't you? Well, how about 10? I think people can just skim many cards at once.

Reply:
okay!